source/_docs/mu-plugin.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Create a WordPress MU Plugin for Actions and Filters
+description: Learn to make a boilerplate MU Plugin for actions and filters.
+contributors:
+ - alexfornuto
+ - eabquina
+ - carlalberto
+---
+
+For actions or filters you want to run even when a theme's `functions.php` isn't invoked by a request, or before plugins are loaded by WordPress, you can create a [**Must-Use** (**MU**) plugin](https://codex.wordpress.org/Must_Use_Plugins){.external}. 
+
+MU-Plugins are activated by default by simply uploading a php file to the `wp-content/mu-plugins` directory. It affects the whole site including all sites under a WordPress Multisite installation. 
+
+It is loaded by PHP, in alphabetical order, before normal plugins. This means API hooks added in an mu-plugin apply to all other plugins even if they run hooked-functions in the global namespace.
+
+## Create Your MU Plugin
+1. Create a PHP file (i.e. your-file.php) in the `mu-plugins` folder. `code/wp-content/mu-plugins/your-file.php`
+2. Provide the plugin details for its name, description etc.
+3. Add the custom functions, along with the filters or action that you want to run.
+
+Use the following script as a starting point for making your own plugin.
+ 
+```php 
+<?php
+/*
+  Plugin Name: Custom Actions and Filters
+  Plugin URI: https://pantheon.io
+  Description: Boilerplate MU-Plugin for custom actions and filters to run for a site instead on setting in WP-config.php
+  Version: 0.1
+  Author: Pantheon
+  Author URI: https://pantheon.io/
+*/
+
+if ( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) ) {
+  // Actions or Filters that will only run in Pantheon .
+};
+
+if ( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) && php_sapi_name() != 'cli' ) {
+	// Add your actions or filters here that you want to exclude during WP CLI execution.
+}
+
+/*
+ * Set $regex_path_patterns accordingly.
+ *
+ * We don't set this variable for you, so you must define it
```

Comment:
In a couple spots we say we don't set the variable and provide an example in comment, then define it below. Is this an oversight or am I not understanding something?